integration/apps/rails-five/Gemfile

```diff
@@ -0,0 +1,117 @@
+require 'datadog/demo_env'
+
+source 'https://rubygems.org' do
+  # Framework dependent gems
+  # gem 'rails', '5.2.2'
+
+  google_protobuf_versions = [
+    '~> 3.0',
+    '!= 3.7.0.rc.2',
+    '!= 3.7.0.rc.3',
+    '!= 3.7.0',
+    '!= 3.7.1',
+    '!= 3.8.0.rc.1'
+  ]
+
+  # gem 'rack', '2.1.4' # Ruby 2.2 compatibility
+  gem 'railties', '5.2.2'
+  gem 'actionpack', '5.2.2'
+  gem 'actionview', '5.2.2'
+  gem 'activemodel', '5.2.2'
+  gem 'activejob', '5.2.2'
+  gem 'activerecord', '5.2.2'
+  gem 'actionmailer', '5.2.2'
+  gem 'actioncable', '5.2.2'
+
+  gem 'mysql2'
+  gem 'puma'
+  gem 'unicorn'
+
+  # Choose correct specs for 'ddtrace' demo environment
+  Datadog::DemoEnv.gem_spec('ddtrace') do |spec|
+    if !spec.empty?
+      gem 'ddtrace', *spec
+    else
+      # Install the latest prerelease by default
+      source 'http://gems.datadoghq.com/prerelease' do
+        gem 'ddtrace', '0.43.0.feature.profiling.98496'
+      end
+    end
+  end
+
+  # Switch to this when profiling goes mainline
+  # gem 'ddtrace', *Datadog::DemoEnv.gem_spec('ddtrace')
+
+  gem 'dogstatsd-ruby'
+  gem 'ffi'
+  gem 'google-protobuf'
+  # gem 'google-protobuf', *google_protobuf_versions # Ruby 2.2 compatibility
+  # gem 'rbtrace'
+  # gem 'ruby-prof'
+
+  # Fixes conflict with profiling (patch overwrite in Thread)
+  # Upgrade this to latest when this patch is merged & released.
+  gem 'rollbar', git: 'https://github.com/delner/rollbar-gem.git', branch: 'thread_prepend'
+
+  # Gems which give aide to higher performance
+  gem 'hiredis', '~> 0.6', platform: :ruby
+  gem 'multi_json'
+  gem 'oj', '3.3', platform: :ruby
+
+  gem 'active_model_serializers', '0.9.3'
+  gem 'activerecord-import' # Speeds up mass imports
+  gem 'aws-sdk', '< 2.0'
+  gem 'bcrypt-ruby', platform: :ruby
+  gem 'connection_pool'
+  gem 'devise'
+  gem 'faker', require: false # Make up fake data to put in models for load testing
+  gem 'geoip'
+  gem 'hawk-auth'
+  gem 'httparty'
+  gem 'ipaddress'
+  # gem 'i18n', '1.5.1' # Ruby 2.2 compatibility
+  # gem 'newrelic_rpm', platform: :ruby
+  gem 'rabl', platform: :ruby
+  gem 'rack-cors'
+  gem 'rake'
+  gem 'redis'#, '< 4.0'
+  gem 'resque'
+  # gem 'resque-pool' # Incompatible with Redis 4.0+
+  gem 'resque-scheduler'
+  gem 'tzinfo-data', platforms: [:mingw, :mswin, :jruby]
+  gem 'validates_timeliness', '~> 3.0.8' # Date comparisons
+  gem 'versionist'
+  gem 'warden'
+
+  group :development do
+    gem 'awesome_print'
+    gem 'annotate'
+    gem 'bullet'
+  end
+
+  group :test do
+    gem 'rspec'
+    gem 'rspec-rails'
+    gem 'rspec-collection_matchers'
+    gem 'database_cleaner'
+    gem 'factory_girl_rails', '4.5.0'
+    gem 'shoulda-matchers', '4.0.1'
+    gem 'timecop'
+    gem 'ci_reporter_rspec'
+    gem 'simplecov', require: false # Will install simplecov-html as a dependency
+    gem 'webmock'
+  end
+
+  gem 'pry' # Ruby 2.2 compatibility
```

Comment:
I think its just the tool we've been using for debugging; it works for all versions of Ruby we support, and we do need to use it from time to time. Maybe we can replace with `byebug` or something else if it has the same kind of compatibility.

Reply:
I definitely prefer pry, especially because there's [`pry-byebug`](https://rubygems.org/gems/pry-byebug) to give you debugger powers in pry (+ MRI, if you want on JRuby there's [`pry-debugger-jruby`](https://rubygems.org/gems/pry-debugger-jruby)).

My confusion was mostly around the comment; it looks like it's not very relevant -- perhaps remove?